content/04.model-complexity.md

```diff
@@ -1,42 +1,45 @@
+Such bootstrapping of the training data in addition to that included in the model (bagging) helped generate a distribution (and confidence intervals) of the importance scores of the predictive features selected by the model. -->
 
 #### Regularization
 
-Another common strategy for handling the paucity of data in rare disease is to aggregate data from multiple studies or time points to produce a more comprehensive dataset. 
+<!-- TODO: I think of regularization as a way to get simpler models. I'm not sure if the properties described in the first paragraph of this section are things that would be improved by simpler models. It feels like these are more like technical biases that simpler models might be just as good at picking up on. -->
+Another common strategy for handling the paucity of data in rare disease is to aggregate data from multiple studies or time points to produce a more comprehensive dataset.
 Given a dataset with strong preexisting study-specific technical differences between groups of samples, ML methods may model dataset-specific features instead of true biology, leading to high prediction accuracy for training data but poor performance in new test data (an "overfit" model) [@doi:10.1073/pnas.1900654116].
-Minimization of overfitting can be accomplished by cross-validation (to reduce variance in predictions) and regularization (to reduce low bias in models). 
-Regularization makes models less reliant on training data by adding a small penalty (determined by cross-validation), and can minimize overfitting while improving feature selection. 
+Minimization of overfitting can be accomplished by cross-validation (to reduce variance in predictions) and regularization (to reduce low bias in models).
+Regularization makes models less reliant on training data by adding a small penalty (determined by cross-validation), and can minimize overfitting while improving feature selection.
 
-ML models can be regularized using 3 main methods, each with strengths and weaknesses. 
-Ridge regression aims to minimize the magnitude of the features, but cannot remove unimportant features and thus may not be ideal for reducing the feature space. 
-Another method, LASSO regression, works well for selecting few important features since it can minimize the magnitude of some features more than the others[@doi:10.1038/nmeth.4014]. 
-Elastic-net regression is a combination of LASSO and ridge regression [@doi:10.1111/j.1467-9868.2005.00503.x], and helps to select the most useful features, especially in presence of large number of correlated features. 
+<!-- TODO: What about dropout, etc in neural networks. Is it necessary to explain lasso, ridge, etc? If so, should we name drop other techniques too? Is LASSO or ridge more appropriate for rare disease questions? Why or why not? -->
```

Comment:
We assumed that our audience is someone who is new to machine learning, but wants to apply the techniques to their favorite rare disease data. That was the reasoning behind adding minimal explanations for the techniques. Following that logic I would lean towards adding the other techniques :)

Towards the end of this section, our recommendation was that LASSO and elastic-net would be well suited for rare disease questions.

Reply:
tracked in #112

Reply:
Ok - I'm not convinced that you do need to add other regularization methods, but I do think it would be helpful to consider if there is something unique about the regularization approaches that you are presenting that matters in particular for rare diseases. 😁